custom/ewsghana/api.py

```diff
@@ -702,7 +706,7 @@ def web_user_sync(self, ews_webuser):
         return user
 
     def _reassign_number(self, user, connection):
-        v = VerifiedNumber.by_phone(connection.phone_number, include_pending=True)
+        v = VerifiedNumber.by_phone(apply_leniency(connection.phone_number), include_pending=True)
```

Comment:
@kkrampa Thanks for reporting, I just made a PR so that `apply_leniency` is always used in the `by_phone` lookup automatically. Do you forsee any issue with this pr (https://github.com/dimagi/commcare-hq/pull/8768) as it pertains to any logistics/ilsgateway/ewsghana code?